src/components/Atoms/CurvedButton/CurvedButton.jsx

```diff
@@ -0,0 +1,34 @@
+import styles from './CurvedButton.module.css';
+
+const CurvedButton = ({
+  title,
+  type = 'button',
+  greenOrBlack,
+  recipesOrFavorites,
+  onClick = () => {},
```

Comment:
It will be good idea to inform our developer that he is missing on click handler. 💡 

f.ex. :
```
onClick = () => {
    alert('Please fill on click handler');
  },
```

Reply:
Done I guess? 😅